PHP/Coding Style.md

```diff
@@ -0,0 +1,64 @@
+# Coding Style
+
+Unless otherwise mentioned, we follow the [WordPress PHP Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/) and [PSRs](http://www.php-fig.org/psr/).
+
+We also have our own standards for things that aren’t covered by the standards mentioned above, such as namespacing and general project structure.
+
+
+## File Layout
+
+PHP files should **either** declare symbols (classes, functions, etc) **or** run code (function calls, etc), but not both.
+
+Classes should be in their own file, which should not declare any other functions or classes.

```

Comment:
The use of the word `function` is very ambiguous here. think about rephrasing to use the word `method`/`class function` and `global function`.

In the end what we write in the PHP classes themselves as methods/class_functions is `public function function_name() {}`, and we don't want people to use the Standards to justify not writing anything inside a class 😉